doc/manual/source/user_guide/bootcamp.rst

```diff
@@ -31,15 +31,12 @@ clone the repository using Mercurial:
 
```

Comment:
There's a reference to bitbucket that should be updated to gitub.  (Can't figure out how to put the comment further up in the file, sorry about that.)

Reply:
fixed